src/stream/src/task/barrier_manager.rs

```diff
@@ -298,6 +306,16 @@ pub(crate) struct StreamActorManager {
     pub(super) runtime: BackgroundShutdownRuntime,
 }
 
+#[derive(Debug)]
+#[expect(dead_code)]
+pub(super) struct LocalBarrierWorkerDebugInfo<'a> {
+    actor_to_send: HashSet<ActorId>,
+    running_actors: HashSet<ActorId>,
+    creating_actors: Vec<HashSet<ActorId>>,
```

Comment:
It would be better to avoid using `HashSet` or `HashMap` in the output, as the ordering of elements are not guaranteed. What about using `BTree` collections?